ui/app/styles/core/forms.scss

```diff
@@ -73,6 +73,7 @@ label {
 .sub-text {
   color: $grey;
   margin-bottom: 0.25rem;
+  font-size: 12px;
```

Comment:
Here too. If the designs don't match the sizing variables we have we need to revisit with the design team

Reply:
See comment above.